src/app/components/trade/LiquidityDetails.tsx

```diff
@@ -22,7 +22,7 @@ import { DropdownPopper } from 'app/components/Popover';
 import { Typography } from 'app/theme';
 import bnJs from 'bnJs';
 import { BIGINT_ZERO, FRACTION_ONE, FRACTION_ZERO } from 'constants/misc';
-import { BalanceState, useAvailablePairs, useBalances } from 'hooks/useV2Pairs';
+import { BalanceData, useAvailablePairs, useBalances } from 'hooks/useV2Pairs';
```

Comment:
Why did you change the variable name?

Reply:
@lijac

Reply:
It is to ensure consistency, there are types (interfaces actually) `BalanceState`, `BalanceData`, `ReserveState`, `ReserveData`.